apps/dokploy/templates/teable/docker-compose.yml

```diff
@@ -0,0 +1,83 @@
+version: '3.9'
+
+services:
+  teable:
+    image: ghcr.io/teableio/teable:1.3.1-alpha-build.460
+    restart: always
+    ports:
+      - ${TEABLE_PORT}
+    volumes:
+      - teable-data:/app/.assets:rw
+      # you may use a bind-mounted host directory instead,
+      # so that it is harder to accidentally remove the volume and lose all your data!
+      # - ./docker/teable/data:/app/.assets:rw
+    environment:
+      - TZ=${TIMEZONE}
+      - NEXT_ENV_IMAGES_ALL_REMOTE=true
+      - PUBLIC_ORIGIN=${PUBLIC_ORIGIN}
+      - PRISMA_DATABASE_URL=${PRISMA_DATABASE_URL}
+      - PUBLIC_DATABASE_PROXY=${PUBLIC_DATABASE_PROXY}
+      - BACKEND_MAIL_HOST=${BACKEND_MAIL_HOST}
+      - BACKEND_MAIL_PORT=${BACKEND_MAIL_PORT}
+      - BACKEND_MAIL_SECURE=${BACKEND_MAIL_SECURE}
+      - BACKEND_MAIL_SENDER=${BACKEND_MAIL_SENDER}
+      - BACKEND_MAIL_SENDER_NAME=${BACKEND_MAIL_SENDER_NAME}
+      - BACKEND_MAIL_AUTH_USER=${BACKEND_MAIL_AUTH_USER}
+      - BACKEND_MAIL_AUTH_PASS=${BACKEND_MAIL_AUTH_PASS}
+    networks:
+      - dokploy-network
+      - teable-standalone
+    labels:
+      - "traefik.enable=true"
+      - "traefik.http.routers.${HASH}.rule=Host(`${TEABLE_HOST}`)"
+      - "traefik.http.services.${HASH}.loadbalancer.server.port=${TEABLE_PORT}"
+    depends_on:
+      teable-db-migrate:
+        condition: service_completed_successfully
+
+  teable-db:
+    image: postgres:15.4
+    restart: always
+    ports:
+      - '42345:5432'
```

Comment:
Do: 
```suggestion
      - ${POSTGRES_PORT} 
```

Reply:
I guess the application open the 42345 port for external database connection - https://help.teable.io/developer/database-connection, not sure that is a good idea to use default db port for that

Reply:
Got it, but in this case we can have only 1 instance of this template because the port is already allocated

Reply:
added random value for the port

Reply:
Good idea, @kucherenko do you know if this network is required? I deleted and everything seems to work fine
```yaml
networks:
  teable-standalone:
    name: teable-standalone-network
    driver: bridge
 ```

Reply:
updated